ams/server/src/main/java/com/netease/arctic/server/metrics/MetricsReporters.java

```diff
@@ -0,0 +1,16 @@
+package com.netease.arctic.server.metrics;
+
+import com.netease.arctic.ams.api.metrics.MetricReporter;
+import org.apache.iceberg.metrics.MetricsReport;
+
+import java.util.Set;
+
+public class MetricsReporters {
+  private Set<MetricReporter> metricReporters;
+
+  public void report(MetricsReport report) {
+    for (MetricReporter reporter : metricReporters) {
```

Comment:
I think we need to consider the follow two points:
1. Isolate exceptions between `reporters` to avoid mutual influence between `reporters`
2. Different `reporters` require different scheduling cycles.

WDYT?

Reply:
1.Different reporters should be isolated from each other to reduce mutual interference. This code needs to be enriched, including the registration logic of reporters.
2.My idea is that the reporting cycle is controlled internally by the reporter.